mesos-client/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'application'
+}
+
+dependencies {
+    implementation 'com.typesafe.akka:akka-stream_2.12:2.5.19'
+    implementation 'org.apache.mesos:mesos:1.7.1'
+    implementation 'com.typesafe.akka:akka-http_2.12:10.1.7'
+    implementation 'com.lightbend.akka:akka-stream-alpakka-simple-codecs_2.12:0.20'
+
+    runtime 'org.slf4j:slf4j-simple:1.7.25'
```

Comment:
Why is this required? Shouldn't this be a `testRuntimeOnly` dependency?

Reply:
this is required for the main class to run (and to be able to log)

Reply:
Why would one use `gradle run` on this library?